BUILD

```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//visibility:public"])
+
+java_library(
+    name = "grpc_java",
```

Comment:
We had one of these targets internally, and it caused problems, because tons got added to it. If it is _just_ core+stub, that seems like it could be fine. But on the other hand, one vs two targets for deps isn't _that_ bad. @carl-mastrangelo, thoughts?